src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -1260,6 +1277,7 @@ internal void AddDefaultTrustedLocations()
             if (!IsFirstRun) return;
             string ProgramData = Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData);
             AddTrustedLocation(Path.Combine(ProgramData, Configurations.AutodeskAsString));
+            AddTrustedLocation(Path.Combine(ProgramData, Configurations.DynamoAsString));
```

Comment:
I think this is for the Dynamo folder for sandbox right?

Reply:
yes, I think we discussed that we should remove it, will do

Reply:
I personally have no problem adding it, but I thought we left Dynamo folder out for a reason. Maybe @mjkkirschner @saintentropy would know